ui/client/package.json

```diff
@@ -11,78 +11,81 @@
     "test": "NODE_ENV=production ./node_modules/karma/bin/karma start --single-run"
   },
   "dependencies": {
-    "bootstrap": "3.3.6",
-    "classnames": "2.2.5",
-    "file-saver": "1.3.3",
-    "flattenizer": "1.0.4",
+    "bootstrap": "^3.4.1",
```

Comment:
Please use explicit versioning, i.e "3.4.1" instead of "^3.4.1" (which is non-deterministic over time)(https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json)

Reply:
I can agree with that we shouldn't use ^ but ~ it's okey. I will replace ^ to ~.